### PR TITLE
Clarify usage of documentation

### PR DESCRIPTION
**Pull request by JasperCraeghs**
- Removed _README_ from title _Sphinx Traceability plugin_
- Grouped example documentation under a common title 'Example usage'

### CODE REVIEW

File: doc/example_usage.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+==============================
+Example usage
+==============================
```
Comment (Letme) [inline doc/example_usage.rst]: This here is too long - should be a doc warning in travis.

File: README.rst [hunk 2]
```diff
@@ -30,13 +30,17 @@
...
-Sphinx Traceability plugin - README
-===================================
+==========================
+Sphinx Traceability plugin
+==========================
 
 Sphinx plugin that allows defining documentation items and relations between those items. Can be used
 as a requirements management tool for e.g. ISO26262 projects.
 
+.. contents:: `Contents`
```
Comment (Letme) [inline README.rst]: This makes a nice toc indeed, but the title of TOC is strange. Maybe change it to `== Contents ==` ?

Comment (JasperCraeghs) [reply README.rst]: This is the TOC title in every file in _Example usage_. Do you want it changed there as well?

File: doc/example_usage.rst (hunk 1 above)
Comment (JasperCraeghs) [reply doc/example_usage.rst]: 4cadc78dd05210eb5132d97ea88de7ab93c7b870

File: README.rst (hunk 2 above)
Comment (Letme) [reply README.rst]: This is a different beast as it seems at least in markdown. We can check rendered view to see whats going on...

**Pull request merged**